tests/async_engine/test_api_server.py

```diff
@@ -25,23 +25,21 @@ def _query_server_long(prompt: str) -> dict:
 
 
 @pytest.fixture
-def api_server():
+def api_server(num_tokenizer_actors: int):
     script_path = Path(__file__).parent.joinpath(
         "api_server_async_engine.py").absolute()
     uvicorn_process = subprocess.Popen([
-        sys.executable,
-        "-u",
-        str(script_path),
-        "--model",
-        "facebook/opt-125m",
-        "--host",
-        "127.0.0.1",
+        sys.executable, "-u",
+        str(script_path), "--model", "facebook/opt-125m", "--host",
+        "127.0.0.1", "--num-tokenizer-actors",
+        str(num_tokenizer_actors)
     ])
     yield
     uvicorn_process.terminate()
 
 
-def test_api_server(api_server):
+@pytest.mark.parametrize("num_tokenizer_actors", [0, 2])
```

Comment:
This is a good smoke test but note we don't have any test that verifies the tokens are tokenized correctly. I think a unit test at the tokenizergroup level will give us a lot of that value.